### PR TITLE
Add start/end/topic to stats output

### PR DESCRIPTION
**Pull request by galgeek**
hi @whimboo !

This patch adds start, end, etherpad, and topic properties to metrics object when we reset data for a new test day, so that they'll be available to generateStats and saved with metrics, until the next test day begins, addressing my comment in issue #23.

### CODE REVIEW

**Comment by whimboo** [comment]
Not sure why this PR was asked for review already given that it misses most of the requirements as given on issue #23. We may go with this as round 1, but certainly have to follow-up with other PR to cover all the requirements. Please let me know.

**Comment by galgeek** [comment]
Hi @whimboo!

This PR is meant only to address my comment on issue #23, not the entire issue.

Do we want to do something other than send stats via IRC to the requester for round 1?

File: bot.js [hunk 1]
```diff
@@ -46,6 +46,10 @@ function resetData() {
     firebotBugs:[],
     usersTalked: {},
     hourUTC: {},
+    start: startTime.toUTCString(),
+    end: endTime.toUTCString(),
```
Comment (whimboo) [inline bot.js]: So you are adding those items but they are not used in `generateStats`. We might want to print out not only the duration but also the start and end time like: `Test Day run from XYZ till ZYX (YZX hours)`. But for that we would have to refactor this method to not iterate over all keys and print only some, but prints the ones we need without a loop.

Comment (galgeek) [inline bot.js]: Hi @whimboo!

generateStats does currently print all metrics properties. Arrays and objects are handled first and other properties at the end of the block.

Comment (whimboo) [inline bot.js]: Ok, then it is fine. Just wanted to ensure that those newly added properties are really visible. Btw. this PR cannot be merged, and needs an update to the latest code.

**Comment by galgeek** [comment]
Hi @whimboo!

I've rebased and updated the PR.

Thank you for your feedback and help!

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 *.swp
 .DS_Store
-
+config.json.test
+config.json.qa
```
Comment (whimboo) [inline .gitignore]: AGain, this is not part of this PR. Please make sure to check all of your changes before uploading the commit. You can do that with `git diff master`.

**Comment by galgeek** [comment]
Hi @whimboo!

Here's another commit backing out those .gitignore edits.

Thank you!

Barbara

**Comment by whimboo** [comment]
I rebased the commit, fixed the commit message and merged into master as https://github.com/mozilla/testdaybot/commit/00a1f0b1eaab5d9a307bb6495a35752377776177.
